applications/web-browser/page.js

```
@@ -0,0 +1,223 @@
+		var domain = matches && matches[1];
+		return "https://" + domain + url;
+	} else if (url.substring(0, 2) === "./") {
+		url = "." + url;
+	} else if (/^\s*$/.test(url)) {
+		return ""; //Empty = Return nothing
+	} else {
+		url = "../" + url;
+	}
+	url = base_url + url;
+	var i = 0;
+	while (/\/\.\.\//.test((url = url.replace(/[^\/]+\/+\.\.\//g, "")))) {}
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)